core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/RepairStateImpl.java

```diff
@@ -170,17 +170,15 @@ private long calculateRepairedAt(final VnodeRepairStates vnodeRepairStates, fina
 
     private long repairedTableRepairedAt(final long repairedAt, final RepairStateSnapshot old)
     {
-        if (LOG.isInfoEnabled())
+        if (LOG.isDebugEnabled())
```

Comment:
In my view, the biggest improvement here is this change to DEBUG, as this will mean less people see the messages, so less confusion. The message itself is only a slight change.

I'm not familiar enough with this functionality to understand why it is important to print out timestamps that have already passed. Naïvely, for these scenarios, wouldn't something like this be more accurate:

 ... next repair as soon as possible.
 ... did not complete within interval, next repair as soon as possible.

Or whatever the actual scenario is that is happening.